packages/server/graphql/public/types/TimelineEvent.ts

```diff
@@ -0,0 +1,17 @@
+import {GraphQLFieldConfigMap} from 'graphql'
+import TimelineEvent from '../../../database/types/TimelineEvent'
+import {GQLContext} from '../../graphql'
+import {timelineEventInterfaceFields} from '../../types/TimelineEvent'
+
+type TimelineEventSource = TimelineEvent & {orgId: string; teamId: string}
+
+export const timelineEventInterfaceResolvers = () =>
+  Object.fromEntries(
+    Object.entries(
+      timelineEventInterfaceFields() as GraphQLFieldConfigMap<TimelineEventSource, GQLContext>
```

Comment:
+1 You could properly type the function already, so types are properly checked.

Reply:
Moved return type to the function def